--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -13,10 +13,10 @@ import sphinx_rtd_theme
 
 
 def setup(app):
-    app.add_javascript("https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js")
-    app.add_javascript("https://unpkg.com/@jupyter-widgets/html-manager@*/dist/embed-amd.js")
+    app.add_js_file("https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js")
+    app.add_js_file("https://unpkg.com/@jupyter-widgets/html-manager@*/dist/embed-amd.js")
 
-    app.add_stylesheet("main_stylesheet.css")
+    app.add_css_file("main_stylesheet.css")
 
 html_theme = "sphinx_rtd_theme"
 
